index.bs

```diff
@@ -1088,6 +1092,7 @@ The <dfn method for=FileSystemSyncAccessHandle>read(|buffer|, {{FileSystemReadWr
     1. Otherwise set |result| to 0.
 1. Let |arrayBuffer| be |buffer|'s [=underlying buffer=].
 1. [=ArrayBuffer/write|Write=] |bytes| into |arrayBuffer|.
+1. Update the file cursor offset to be |readStart| + |result|.
```

Comment:
Sorry, with "same below" I meant that you'd still have to patch a couple places manually, such as this one and others.

Reply:
Ah, thanks